### DISCUSSION TITLE
win 11 update

### DISCUSSION THREAD

**Discussion by ra8384** [microsoft/winget-dsc#139] [General]
Omg, this is so much a long-awaited feature for me.

I maintain an open-source (actually, fair source) project and I have multiple repositories for different services (server & agent software).

I am used to creating all tickets on the primary repo (server) and then, if there is work to be done for an agent, I create an issue in the agent's repo with a single line: "Relates to ####".

Now I can create sub-issues for another repo in my "epics" and better see the actual progress!

**Comment by ra8384** [comment]
It seems like you're encountering an issue with the `Microsoft.VSCode.Dsc` module not detecting the presence of Visual Studio Code on your Windows 11 ARM virtual machine. This issue has been reported by other users as well.

The problem might be due to a mismatch in the registry key values used to detect the installation of VSCode. The `Microsoft.VSCode.Dsc` module checks for specific registry keys to verify the presence of VSCode, and these keys can vary between different system architectures and installation methods.

Here are a few steps you can try to resolve this issue:
1. **Check Registry Keys**: Verify the registry keys on your system to ensure they match the ones expected by the `Microsoft.VSCode.Dsc` module. You can find the expected keys in the module's source code or documentation.
2. **Update the Module**: Ensure you are using the latest version of the `Microsoft.VSCode.Dsc` module. There might be updates or patches that address this issue.
3. **Manual Installation**: As a workaround, you can manually install the required VSCode extensions and verify their presence before running the configuration script.

If these steps don't resolve the issue, you might want to follow the discussion on the [GitHub issue page](https://github.com/microsoft/winget-dsc/issues/105) for more detailed troubleshooting and potential fixes from the community.

Feel free to ask if you need further assistance or have any other questions!

**Reply by Gijsreyn** [reply]
It should be solved in #135.
